js_modules/dagster-ui/packages/ui-core/src/asset-graph/SidebarAssetInfo.tsx

```diff
@@ -34,21 +35,16 @@ import {buildRepoAddress} from '../workspace/buildRepoAddress';
 import {RepoAddress} from '../workspace/types';
 import {workspacePathFromAddress} from '../workspace/workspacePath';
 
-import {
-  LiveDataForNode,
-  displayNameForAssetKey,
-  GraphNode,
-  nodeDependsOnSelf,
-  stepKeyForAsset,
-} from './Utils';
+import {displayNameForAssetKey, GraphNode, nodeDependsOnSelf, stepKeyForAsset} from './Utils';
 import {SidebarAssetQuery, SidebarAssetQueryVariables} from './types/SidebarAssetInfo.types';
 import {AssetNodeForGraphQueryFragment} from './types/useAssetGraphData.types';
 
 export const SidebarAssetInfo: React.FC<{
   graphNode: GraphNode;
-  liveData?: LiveDataForNode;
-}> = ({graphNode, liveData}) => {
+}> = ({graphNode}) => {
   const {assetKey, definition} = graphNode;
+  const {liveDataByNode} = useAssetsLiveData(React.useMemo(() => [assetKey], [assetKey]));
```

Comment:
Technically this shouldn't be necessary right? (Just imagining since the useAssetsLiveData hook does a bunch of internal fanciness and cache control)

This will impact a lot of callsites so it'd be nice to push it down into the hook

Reply:
Yeah I made another hook for the case of a single key, will update the pr